scripts/api.js

```diff
@@ -0,0 +1,29 @@
+// recuperatation des donnees sources
+export async function getDataPhotographers() {
+  const response = await fetch("./data/photographers.json");
+  const dataFisheye = await response.json();
+  console.log(dataFisheye);
+  const dataPhotographers = [...dataFisheye.photographers];
+  const dataMedias = [...dataFisheye.media];
+  return {
+    dataPhotographers,
+    dataMedias,
+  };
```

Comment:
```suggestion
  return {
    dataPhotographers: dataFisheye.photographers ,
    dataMedias: dataFisheye.media,
  };
```

Même si je ne comprend pas l'intêret du tout de ces lignes de code, mis à part que tu changes le noms des propriétés